server/modules/record/manager/recordManager.js

```diff
@@ -51,10 +63,19 @@ const fetchRecordAndNodesByUuid = async (surveyId, recordUuid, draft = true, cli
   return Record.assocNodes(ObjectUtils.toUuidIndexedObj(nodes))(record)
 }
 
+/**
+ * Creates a Node object given an array of values (used for nodes batch insert)
+ */
+const _fromNodeValuesToNode = nodeValues =>
+  NodeRepository.nodeTableColumns.reduce(
+    (accContent, key, index) => Object.assign(accContent, { [camelize(key)]: nodeValues[index] }),
+    {}
+  )
+
```

Comment:
I would inline this function. it's used only in insertNodesFromValues

Reply:
ok, you are right